### ISSUE TITLE
Stringify then Parse

### ISSUE THREAD

**Issue by amykuner** [fac22/week6-basc#55]
I could be completely wrong here because I haven't checked the outputs of each, but if you're stringifying your data when it comes from your database, and then parsing the data you've just stringified inside your component, could you not just skip both steps as they're the inverse of each other?

https://github.com/fac22/week6-basc/blob/1a064e723d5f54bded77647d6a3871529124fa2b/pages/index.js#L10

**Comment by oliverjam** [comment]
Yeah I'm pretty sure Next automatically stringifies all your server-side props in order to send them down to the client, so you shouldn't need to do this manually
